test/cpp/jit/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ elseif (USE_ROCM)
 
   target_link_libraries(test_jit PRIVATE caffe2_gpu)
 
+  target_link_libraries(test_jit PRIVATE torch_cpu)
```

Comment:
What is the purpose of this?

Reply:
it is trying to resolve link error, but it seems not to resolve the failures in CI